pkgs/by-name/ve/ventoy/package.nix

```diff
@@ -96,12 +92,14 @@ stdenv.mkDerivation (finalAttrs: {
     which
     xz
   ]
-  ++ optional withCryptsetup cryptsetup
-  ++ optional withExt4 e2fsprogs
-  ++ optional withGtk3 gtk3
-  ++ optional withNtfs ntfs3g
-  ++ optional withXfs xfsprogs
-  ++ optional withQt5 qtbase;
+  ++ optionals withCryptsetup [ cryptsetup ]
+  ++ optionals withExt4 [ e2fsprogs ]
+  ++ optionals withGtk3 [ gtk3 ]
+  ++ optionals withNtfs [ ntfs3g ]
+  ++ optionals withXfs [ xfsprogs ]
+  ++ optionals withQt5 [ qtbase ];
```

Comment:
Just out of curiosity, what is wrong with "optional"? Does it mean we should avoid using it for some reason?

Reply:
Nothing, to be honest. It is personal preference and comfort.
I do not like some mental overloads: `optional` is limited to a single element, and I never remember if it needs the `[]` brackets.

Reply:
I see)